testdata/vast4.2_click_through_in_wrapper.xml

```diff
@@ -0,0 +1,90 @@
+            <Pricing model="cpm" currency="USD">
+                <![CDATA[ 25.00 ]]>
+            </Pricing>
+            <AdTitle>iabtechlab video ad</AdTitle>
+            <Category authority="http://www.iabtechlab.com/categoryauthority">AD CONTENT description category</Category>
+            <Creatives>
+                <Creative id="5480" sequence="1" adId="2447226">
+                    <UniversalAdId idRegistry="Ad-ID" idValue="8465">8465</UniversalAdId>
+                    <UniversalAdId idRegistry="Ad-2" idValue="1234">1111</UniversalAdId>
+                    <Linear>
+                        <TrackingEvents>
+                            <Tracking event="start" offset="08:34:01">http://example.com/tracking/start</Tracking>
+                            <Tracking event="firstQuartile">http://example.com/tracking/firstQuartile</Tracking>
+                            <Tracking event="midpoint">http://example.com/tracking/midpoint</Tracking>
+                            <Tracking event="thirdQuartile">http://example.com/tracking/thirdQuartile</Tracking>
+                            <Tracking event="complete">http://example.com/tracking/complete</Tracking>
+                        </TrackingEvents>
+                        <Duration>00:00:16</Duration>
+                        <MediaFiles>
+                            <MediaFile id="5241" delivery="progressive" type="video/mp4" bitrate="2000" width="1280"
+                                       height="720" minBitrate="1500" maxBitrate="2500" scalable="1"
+                                       maintainAspectRatio="1" codec="0">
+                                <![CDATA[https://iabtechlab.com/wp-content/uploads/2016/07/VAST-4.0-Short-Intro.mp4]]>
+                            </MediaFile>
+                            <MediaFile id="5244" delivery="progressive" type="video/mp4" bitrate="1000" width="854"
+                                       height="480" minBitrate="700" maxBitrate="1500" scalable="1"
+                                       maintainAspectRatio="1" codec="0">
+                                <![CDATA[https://iabtechlab.com/wp-content/uploads/2017/12/VAST-4.0-Short-Intro-mid-resolution.mp4]]>
+                            </MediaFile>
+                            <MediaFile id="5246" delivery="progressive" type="video/mp4" bitrate="600" width="640"
+                                       height="360" minBitrate="500" maxBitrate="700" scalable="1"
+                                       maintainAspectRatio="1" codec="0">
+                                <![CDATA[https://iabtechlab.com/wp-content/uploads/2017/12/VAST-4.0-Short-Intro-low-resolution.mp4]]>
+                            </MediaFile>
+                        </MediaFiles>
+                        <VideoClicks>
+                            <ClickThrough id="blog">
+                                <![CDATA[https://iabtechlab.com]]>
```

Comment:
use .test tlds

Reply:
if the tests doesn't need to validate the data/responses from the endpoints, it's better to not use domains of third parties.